src/item_pocket.cpp

```diff
@@ -1706,6 +1706,54 @@ void item_pocket::process( map &here, Character *carrier, const tripoint &pos, f
     }
 }
 
+bool item_pocket::leak( map &here, Character *carrier, const tripoint &pos, item_pocket *pocke )
+{
+    std::vector<item *> erases;
+    for( auto iter = contents.begin(); iter != contents.end(); ) {
+        if( iter->leak( here, carrier, pos, this ) ) {
+            if( watertight() ) {
+                continue;
+            }
+            item *it = &*iter;
+            int count = it->count();
+
+            if( pocke ) {
+                if( pocke->watertight() ) {
+                    ++iter;
+                    continue;
+                }
+                if( count > 1 && it->has_flag( flag_MELTING_FROZEN ) && false ) {
```

Comment:
The `&& false` makes the whole condition false regardless of the other terms. Same for the condition below.

Reply:
I forgot it.
This meaning is allowed to only leak all at once at the moment for temporary, because if we drip it little by little from container to container, the error "liquid and the other stuff are mixed" raised occurs frequently(when save game loaded).
I would like to rewrite this point after a bit of thinking.
Thank you.

Reply:
The flag MELTING_FROZEN and dripping method with little by little was removed.